packages/fhir-team-management/src/components/TeamsAddEdit/index.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useState } from 'react';
+import { Helmet } from 'react-helmet';
+import { Organization, Practitioner, PractitionerRole } from '../../types';
+import Form, { FormField } from './Form';
+import { useParams } from 'react-router';
+import { PRACTITIONERROLE_GET, PRACTITIONER_GET, TEAMS_GET } from '../../constants';
+import { sendErrorNotification } from '@opensrp/notifications';
+import { Spin } from 'antd';
+import lang from '../../lang';
+import { useQuery } from 'react-query';
+import { FHIRServiceClass } from '@opensrp/react-utils';
+import { loadTeamPractitionerInfo } from '../../utils';
+
+export interface Props {
+  fhirBaseURL: string;
+}
+
+export const TeamsAddEdit: React.FC<Props> = (props: Props) => {
+  const { fhirBaseURL } = props;
+
+  const practitionerAPI = new FHIRServiceClass<Practitioner>(fhirBaseURL, 'Practitioner');
+  const organizationAPI = new FHIRServiceClass<Organization>(fhirBaseURL, 'Organization');
+  const practitionerroleAPI = new FHIRServiceClass<PractitionerRole>(
+    fhirBaseURL,
+    'PractitionerRole'
+  );
+  const params: { id?: string } = useParams();
+  const [initialValue, setInitialValue] = useState<FormField>();
+
+  const Practitioners = useQuery(PRACTITIONER_GET, async () => practitionerAPI.list(), {
```

Comment:
add pagination params, the response here will only return 20 practitioners

Reply:
This Component is not to show Practitioners 😅 i need whole list of these to filter then later alternatively we can instead of fetching all we can make server request and fetch only specific one but we are using caching here it wount be much effective against single resource fetch